portal/index.html

```diff
@@ -11,6 +11,12 @@
       window.global ||= window;
     </script>
 
+    <!-- Google Fonts -->
+    <link
```

Comment:
Instead of loading these in the index.html, we should include the required font files in the `assets/fonts` directory and define the font faces in `assets/styles/font.css`. This will allow the files to be loaded asynchronously, instead of having the page load them at the start.